remote-trait-object/src/service/serde_support.rs

```diff
@@ -101,74 +101,81 @@ impl<'de, P: ?Sized + Service, T: ToRemote<P>> Deserialize<'de> for ServicePoint
 
 #[cfg(test)]
 mod tests {
-    mod mock {
-        use super::super::port_thread_local;
-        use crate::port::null_weak_port;
-
-        pub fn set_global_port() {
-            port_thread_local::set_port(null_weak_port());
-        }
-    }
-
     mod serialize_test {
         use super::super::SArc;
-        use super::mock;
-        use crate::{ToDispatcher, HandleToExchange, Port, Service, Dispatch};
+        use crate::service::ServiceObjectId;
+        use crate::*;
+        use std::sync::atomic::{AtomicU32, Ordering};
         use std::sync::{Arc, Weak};
 
-        trait Foo: Service {
-            fn get_handle_to_exchange(&self) -> HandleToExchange;
+        #[derive(Debug)]
+        pub struct MockPort {
+            count: AtomicU32,
         }
 
-        struct FooImpl {
-            pub handle_to_exchange: HandleToExchange,
-        }
-        impl FooImpl {
-            pub fn new(handle: u32) -> Self {
-                Self {
-                    handle_to_exchange: HandleToExchange(handle),
-                }
+        impl Port for MockPort {
+            fn call(&self, _packet: PacketView) -> Packet {
+                unimplemented!()
             }
-        }
-        impl Foo for FooImpl {
-            fn get_handle_to_exchange(&self) -> HandleToExchange {
-                self.handle_to_exchange
+
+            fn register(&self, _service_object: Arc<dyn Dispatch>) -> HandleToExchange {
+                self.count.fetch_add(1, Ordering::SeqCst);
+                HandleToExchange(123)
```

Comment:
I don't understand why you created this commit: "Fix the test which depends on export detail". Could you explain it to me?